.vscode/settings.json

```diff
@@ -4,9 +4,13 @@
 
```

Comment:
The formatting consistency with `autopep8` is excellent, @m3mayllon. 
Thanks for activating auto-testing, too.